doc/client_lua_api.md

```diff
@@ -66,35 +65,25 @@ On an installed version on Linux:
 
 Modpack support
 ----------------
+**NOTE: Not implemented yet.**
+
 Mods can be put in a subdirectory, if the parent directory, which otherwise
 should be a mod, contains a file named `modpack.txt`. This file shall be
 empty, except for lines starting with `#`, which are comments.
 
 Mod directory structure
 ------------------------
 
-    mods
-    |-- modname
-    |   |-- depends.txt
-    |   |-- screenshot.png
-    |   |-- description.txt
-    |   |-- settingtypes.txt
-    |   |-- init.lua
-    |   |-- models
-    |   |-- textures
-    |   |   |-- modname_stuff.png
-    |   |   `-- modname_something_else.png
-    |   |-- sounds
-    |   |-- media
-    |   `-- <custom data>
-    `-- another
-
+    clientmods
+    ├── modname
+    |   ├── depends.txt
+    |   ├── init.lua
+    └── another
```

Comment:
`mod.conf` works.
`screenshot.png` and `description.png` should stay there since they could be used later and if anyone looks into the mod and sees a description or screenshot he will be happy about it.